### DISCUSSION TITLE
flux realism sampler not showing and image disappears once generated with latest update

### DISCUSSION THREAD

**Discussion by finkbom** [lllyasviel/stable-diffusion-webui-forge#1731] [General]
flux realism sampler not showing and image disappears once generated with latest update -

**Comment by aggregate15** [comment]
I have used it a few times, it takes a little longer, but no issues otherwise. the images are generating normally like any other working sampler (Euler, IPNDM, etc). Try reinstalling Forge or removing any flags or disabling any integrated options. also make sure you are using Schedule type 'Simple'.

**Reply by finkbom** [reply]
ok thanks will do---edit- that fixed it
